deployments/dockerfiles/api.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15
+FROM golang:1.16
```

Comment:
Not sure if we need to fix on this specific version, could just allow for latest image.